examples/cluster-configs/trn1-16-nodes-pcluster.md

```diff
@@ -92,10 +98,16 @@ Where
 
 `suppress-validators` is used here to generalize this command so it will not run into error triggered by tagging policies, if any.
 
-This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console.
+This will create a ParallelCluster in your AWS account, and you may inspect the progress in AWS CloudFormation console. 
 
+You may also check cluster status using `pcluster` command, for example: 
 
-3. After the cluster is created successfully, perform the [post-install actions](../general/ami/post_installation.md) to configure the head node and compute nodes. 
+`pcluster describe-cluster -r us-west-2 -n My-ParallelCluster-Trn1`
+
+
+3. After the cluster is created successfully, post-install actions happen as a part of the cluster creating process. Post-install actions now takes place automatically via `CustomActions` indicated in `launch.yaml` to configure the head node and any static compute nodes (`MinCount` > 0). For your information, `CustomActions` will install Neuron drivers and runtime, EFA drivers (if multi-instance training), and Neuron tools. 
+
+After this step is complete, the ParallelCluster environment is properly configured to run SLURM jobs.
```

Comment:
"After the cluster is created, the ParallelCluster environment is properly configured to run SLURM jobs."

Reply:
done, see https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/14/commits/b00979c0c777f816efc83ed8e2234477bbc8a05f